packages/storybook/stories/search-global.stories.tsx

```diff
@@ -35,3 +35,18 @@ export const WithReset: Story = () => {
     );
 };
 WithReset.parameters = rawCodeParameters;
+
+export const WithForwardRef: Story = () => {
+    const [value, setValue] = useState('');
+    const ref = useRef<HTMLInputElement>();
```

Comment:
Pour faire passer le CI:
```suggestion
    const ref = useRef<HTMLInputElement>(null);
```